L/libLAS/build_tarballs.jl

```diff
@@ -0,0 +1,82 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "libLAS"
+version = v"0.1.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/libLAS/libLAS.git", "e6a1aaed412d638687b8aec44f7b12df7ca2bbbb"),
+    DirectorySource("./bundled")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+
+cd $WORKSPACE/srcdir/libLAS/
+
+atomic_patch -p1 ${WORKSPACE}/srcdir/patches/comment-out-FixupOrdering.patch
+
+if [[ "${target}" == *-mingw* ]]; then
+    atomic_patch -p1 ${WORKSPACE}/srcdir/patches/mingw-rename-libs.patch
+fi
+
+if [[ "${target}" == aarch64-linux-musl* ]] || [[ "${target}" == arm-linux-musleabihf ]]; then
```

Comment:
```suggestion
elif [[ "${target}" == aarch64-linux-musl* ]] || [[ "${target}" == arm-linux-musleabihf ]]; then
```

Reply:
Also, I assume the patch below is for all ARM architectures (so potentially also non linux)?  If so, you can do
```sh
[[ "${proc_family}" == "arm" ]]
```
https://docs.binarybuilder.org/stable/build_tips/#Automatic-environment-variables

Reply:
This was only an issue for `armv7l-linux-musleabihf`, `armv7l-linux-gnueabihf` passed CI without the patch (somehow... IDK man...)